yarn.lock

```diff
@@ -13976,6 +13976,14 @@ react-native-svg@^11.0.0:
     css-select "^2.1.0"
     css-tree "^1.0.0-alpha.39"
 
+react-native-svg@^12.1.0:
+  version "12.1.0"
+  resolved "https://registry.yarnpkg.com/react-native-svg/-/react-native-svg-12.1.0.tgz#acfe48c35cd5fca3d5fd767abae0560c36cfc03d"
+  integrity sha512-1g9qBRci7man8QsHoXn6tP3DhCDiypGgc6+AOWq+Sy+PmP6yiyf8VmvKuoqrPam/tf5x+ZaBT2KI0gl7bptZ7w==
+  dependencies:
+    css-select "^2.1.0"
+    css-tree "^1.0.0-alpha.39"
```

Comment:
Okay, so I added react-native-svg to apps/android; as the dependency from fluent-tester doesn't get linked otherwise. So, this added the [v12.1.0](https://www.npmjs.com/package/react-native-svg/v/12.1.0) entry to yarn.lock; I have now upgraded react-native-svg in apps/fluent-tester and apps/win32 from where 11.0.1 is hoisted, and tested it to be working after the upgrade in android, win32 and ios (I guess Podfile.lock also gets updated but not adding that change here )